lib/config/options/index.ts

```diff
@@ -2253,6 +2253,107 @@ const options: RenovateOptions[] = [
     cli: false,
     env: false,
   },
+  {
+    name: 'jsonataManagers',
+    description: 'Custom managers using JSON queries.',
+    type: 'array',
+    subType: 'object',
+    default: [],
+    stage: 'package',
+    cli: true,
+    mergeable: true,
+  },
+  {
+    name: 'matchQueries',
+    description:
+      'JSON query to use. Valid only within a `jsonataManagers` object.',
+    type: 'array',
+    subType: 'string',
+    parent: 'jsonataManagers',
+    cli: false,
+    env: false,
+  },
+  {
+    name: 'depNameTemplate',
```

Comment:
I don't think it's going to work to define options like `depNameTemplate` twice

Reply:
Have you noticed the change in https://github.com/renovatebot/renovate/blob/c139973798a13e76c4f2bfe52e066e0fa2964004/lib/config/validation.ts#L24? Because that is what allow different managers to have the same option.
 
I imagine it would also be possible to change `parent` so as it accepts one or many parents.